includes/settings/settings-panel.php

```diff
@@ -0,0 +1,369 @@
+<?php
+
+namespace HelloElementor\Includes\Settings;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Settings_Panel {
+
+	public $menu_hook = '';
+
+	public $settings_group = 'hello_elementor_settings';
+
+	public function __construct() {
+
+		add_action( 'admin_menu', [ $this, 'register_settings_page' ], 10 );
+		add_action( 'init', [ $this, 'register_settings' ], 10 );
```

Comment:
Why did you use `10` params? It's a default value.

Reply:
Fixed. Thank you.